sdf-schema.json

```diff
@@ -306,6 +313,26 @@
         "pattern": {
           "type": "string"
         },
+        "format": {
```

Comment:
This is where HJSON would be helpful; having difficulty seeing "forest" for "trees" here. What is "format" modifying, "$ref"?

Reply:
Format is an additional constraint that is part of JSON schema, and used in the same way, to constrain the string item to one of several predefined semantic types e.g. URI, DATETIME, etc.